src/Task/PhpStan/README.md

```diff
@@ -0,0 +1,31 @@
+# phpstan
+
+Check Drupal code against deprecation rules.
+
+### grumphp.yml (with current defaults):
+````yml
+parameters:
+    tasks:
+        phpstan:
+            ignore_patterns: 
+                - '/vendor/'
+                - '/node_modules/'
+                - '/core/'
+                - '/libraries/'
+            extensions: ['php', 'inc', 'module', 'install', 'theme']
+            run_on: ['.']
+            autoload_file:
+              defaults: ~
+              allowed_types: ['string', 'null']
+            configuration:
+              defaults: 'vendor/wunderio/code-quality/config/phpstan.neon'
```

Comment:
It looks like we should by default assume that this configuration file is copied to execution root otherwise include files can not be resolved with defaults: `File 'vendor/wunderio/code-quality/config/vendor/mglaman/phpstan-drupal/extension.neon' is missing or is not readable.`

Reply:
So this should be defaulted to just `phpstan.neon`

Reply:
Defaulted to phpstan.neon